server/src/models/holds/holdInput.ts

```diff
@@ -0,0 +1,6 @@
+
+export interface HoldInput {
+    userID: number;
+    description: string;
+    active: boolean;
+}
```

Comment:
Can we put both this and the `Hold` interface in `holdsSchema.ts` instead, like we're doing with the users schema?